src/blocks/plugins/conditions/edit.js

```diff
@@ -250,6 +250,10 @@ const Edit = ({
 			attrs.meta_compare = 'is_true';
 		}
 
+		if ( 'queryString' === value ) {
```

Comment:
Just a note that I've added the default value here. It was not working without it unless the user first changes the match value from one to another.